### DISCUSSION TITLE
Information About V6

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#4180] [General]
Dear AdonisJS Team,

I hope this message finds you well. I am currently an instructor on Udemy, offering a comprehensive full-stack development course focusing on AdonisJS. As I plan to expand the course content to include a module on RBAC (Role-Based Access Control) implementation using AdonisJS, I have recently come across some tweets mentioning AdonisJS version 6. This has raised concerns for me, as I want to ensure that the course remains up-to-date and relevant for my students.

Therefore, I would greatly appreciate it if you could provide me with an overview of the anticipated release date for AdonisJS version 6. Having this information would enable me to make an informed decision regarding the inclusion of the RBAC module in the current course version. I want to avoid any potential complications or breaking changes that may arise if version 6 is released before or during the development and teaching process.

Thank you very much for your time and assistance. I eagerly await your response.

Best regards, 
Fenil Jariwala
Udemy Instructor

**Comment by feniljariwala82** [comment]
@thetutlage anything on this?

**Comment by McSneaky** [comment]
v6 is currently unknown time far away
I don't have any information about it, but just looking at #releases channel in Discord. It seems like it's going to happen sometime this year
Seems like new core is kinda there, but not all the supporting packages, like Lucid, Auth, Mail, Shield, Repl etc. Virk told in one stream, that he doesn't know the date either, that's why he doesn't want to give release date, coz then people will be like: "But you told it will be ready before xyz time!"

I think it's still months away. Even if core is 100% ready, there's still plenty of other official packages. So maybe sometime mid-to-end of fall? But don't take it as serious timeline or anything. It's just my speculation on it

**Reply by feniljariwala82** [reply]
Thank you for your reply. I got your point. Thank you.
